server/src/main/java/org/opensearch/index/SegmentReplicationPressureService.java

```diff
@@ -272,7 +272,7 @@ public void onFailure(Exception e) {
 
         @Override
         protected String getThreadPool() {
-            return ThreadPool.Names.GENERIC;
+            return ThreadPool.Names.SEGMENT_REPLICATION;
```

Comment:
This component is used during ingest to determine if pressure should be applied, not to perform segrep activities.  I don't think we should use the pool here.

Reply:
yes makes sense, will update it